components/Tabs/src/MDCTabBar.h

```diff
@@ -84,6 +84,31 @@ IB_DESIGNABLE
 /** Ink color for taps on tab bar items. Default: Semi-transparent white. */
 @property(nonatomic, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
 
+/**
+ Display font used for selected item titles.
+ Default is the regular font from the font loader.
+ */
+@property(nonatomic, strong, null_resettable) UIFont *selectedItemTitleFont UI_APPEARANCE_SELECTOR;
```

Comment:
Are Tabs intended to support with Dynamic Type?

Reply:
They don't currently and that's not in scope for this work. It's a pretty hairy problem for tabs, since (among other things) tab bars affect the height of headers.

It seems like something that should be on the roadmap as its own project, though.

Reply:
We are looking at how to support custom fonts (with Dynamic Type) across all of our components.
Do you have a few minutes to discuss offline?

Reply:
I switched them all to text attributes, which I think strikes the right balance here and provides useful customization axes.